flash/core/data/types.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Dict, Optional, Tuple, Type, Union
+
+from pytorch_lightning.utilities.enums import LightningEnum
+
+import flash
+
+INPUT_TRANSFORM_TYPE = Optional[
```

Comment:
We already have an `INPUT_TRANSFORM_TYPE` in `flash/core/utilities/types.py`. Which one should be used?

Reply:
Done ! I didn't see the file. Thanks.